### ISSUE TITLE
DiffPool tutorial does not work

### ISSUE THREAD

**Issue by lisiq** [AntonioLonga/PytorchGeometricTutorial#9]
Thank you for making the videos and notebooks available! They are very nice and helpful.
I saw that the DiffPool model still does not work for the version that is uploaded here. 
I was wondering if you already have the working model available?

Thank you in advance!

**Comment by JosephGatto** [comment]
Bumping this, also not able to get DiffPool tutorial to run! I am using latest Colab version.

**Comment by AntonioLonga** [comment]
Hi guys! 

sorry for the late reply.

Let me try to fix it as soon as possible.

Best Antonio

**Comment by lisiq** [comment]
@AntonioLonga my solution was to remove the BatchNorm1d layers and it worked, I am not sure why with them it wasn't working.

Thanks for checking it again!

**Comment by AntonioLonga** [comment]
I've used the solution by @lisiq Thanks for it :)
